### ISSUE TITLE
How to disable cache?

### ISSUE THREAD

**Issue by sesteva** [cvburgess/SQLDataSource#39]
Thank you for sharing this module. 
I'm writing tests for Datasource extending it and I've found the cache getting in the way.
I'm mocking knex using jest-mock-knex to return different responses.
The challenge is the query is the same, therefore the key in the cache is the same and the object is returned. 
I've tried passing .cache(0) so as to prevent caching at all but this did not work.
Please advise. If there is a potential change in the module needed, I would be glad to collaborate with a PR.
Thanks

**Comment by cvburgess** [comment]
Yeah this would have to be added - please give it a go and open a PR!

One of the main benefits of this lib is caching so I have not invested any resouces into a cache-free scenario but it makes sense to me :)
